src/external/libpcre.m4

```diff
@@ -1,13 +1,45 @@
 AC_SUBST(PCRE_LIBS)
 AC_SUBST(PCRE_CFLAGS)
 
-PKG_CHECK_MODULES([PCRE], [libpcre], [found_libpcre=yes], [found_libpcre=no])
-PKG_CHECK_EXISTS(libpcre >= 7,
-                 [AC_MSG_NOTICE([PCRE version is 7 or higher])],
-                 [AC_MSG_NOTICE([PCRE version is below 7])
-                  AC_DEFINE([HAVE_LIBPCRE_LESSER_THAN_7],
-                            1,
-                            [Define if libpcre version is less than 7])])
+PKG_CHECK_MODULES(
+    [PCRE],
+    [libpcre],
+    [
+        found_libpcre=yes
+        PKG_CHECK_EXISTS(
+            libpcre >= 7,
+            [AC_MSG_NOTICE([PCRE version is 7 or higher])],
+            [
+                AC_MSG_NOTICE([PCRE version is below 7])
```

Comment:
This is of course unrelated to this patch, but I wonder if we could (as another patch/PR) drop support for the old pcre versions.

Reply:
It can be done with simple, pcre[1,2] is now separated to this file + sss_regexp.[ch]
But for some distribution it doesn't make sense for now.